### DISCUSSION TITLE
Neo store reports Mull has vulnerabilities; legit?

### DISCUSSION THREAD

**Discussion by ludespeed** [Divested-Mobile/Mull-Fenix#257] [Q&A] [answered]
Anyone have information on this please?
![Screenshot_20241026_100731_Neo Store](https://github.com/user-attachments/assets/ac59fea1-f2f5-4372-b020-710ee34c884a)

**Comment by SkewedZeppelin** [comment, marked as answer]
duplicate of https://github.com/Divested-Mobile/Mull-Fenix/issues/255 and https://github.com/Divested-Mobile/Mull-Fenix/issues/254 and https://github.com/Divested-Mobile/Mull-Fenix/issues/256
